pynetbox_data_uploader/lib/utils/dataclass_data.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+
+
+# pylint: disable = R0902
+@dataclass
+class Device:
+    """
+    This class instantiates device objects with the device data.
+    """
+
+    tenant: str
+    device_role: str
+    manufacturer: str
+    device_type: str
+    status: str
+    site: str
+    location: str
+    rack: str
+    face: str
+    airflow: str
+    position: str
+    name: str
+    serial: str

```

Comment:
Are any of these attribute optional? Can we set some of these to a default value?

Reply:
Some are optional, none have a default value.

Reply:
if some are optional the default value should be None - you'll get an error if the user doesn't provide these. Set these values to None in the dataclass